cmd/skopeo/layers.go

```diff
@@ -9,13 +14,45 @@ var layersCmd = cli.Command{
 	Name:  "layers",
 	Usage: "get images layers",
 	Action: func(c *cli.Context) error {
-		img, err := parseImage(c)
+		rawSource, err := parseImageSource(c, c.Args()[0])
 		if err != nil {
 			return err
 		}
-		if err := img.LayersCommand(c.Args().Tail()...); err != nil {
+		src := image.FromSource(rawSource)
+		layers := c.Args().Tail()
+		if len(layers) == 0 {
+			ls, err := src.LayerDigests()
+			if err != nil {
+				return err
+			}
+			layers = ls
+		}
+		tmpDir, err := ioutil.TempDir(".", "layers-")
```

Comment:
the only notable change is here - I don't believe we should output somewhere significant but it would be nice to at least log where this is going to write layers and manifest? wdyt?